src/main/java/org/isf/medicalinventory/gui/InventoryEdit.java

```diff
@@ -0,0 +1,672 @@
+			suggestion.setForeground(Color.GRAY);
+			suggestion.setHorizontalAlignment(JLabel.CENTER);
+			suggestion.changeAlpha(0.5f);
+			suggestion.changeStyle(Font.BOLD + Font.ITALIC);
+			searchTextField.getDocument().addDocumentListener(new DocumentListener() {
+				@Override
+				public void insertUpdate(DocumentEvent e) {
+					ajustWidth();
+				}
+
+				@Override
+				public void removeUpdate(DocumentEvent e) {
+					ajustWidth();
+				}
+
+				@Override
+				public void changedUpdate(DocumentEvent e) {
+					ajustWidth();
+				}
+			});
+			searchTextField.setEnabled(false);
+			if (inventory != null) {
+				searchTextField.setEnabled(true);
+			} else {
+				searchTextField.setEnabled(false);
+			}
+		}
+		return searchTextField;
+	}
+
+	private JLabel getDateInventoryLabel() {
+		if (dateInventoryLabel == null) {
+			dateInventoryLabel = new JLabel(MessageBundle.getMessage("angal.common.date.txt"));
+		}
+		return dateInventoryLabel;
+	}
+	
+	private JButton getMoreDataBtn() {
+		if (moreData == null) {
+			moreData = new JButton("...");
+			moreData.setPreferredSize(new Dimension(20, 20));
+			moreData.setEnabled(false);
+		}
+		return moreData;
+	}
+
+	private JTextField getCodeTextField() {
+		if (codeTextField == null) {
+			codeTextField = new JTextField();
+			if (inventory != null) {
+				codeTextField.setEnabled(false);
+			} else {
+				codeTextField.setEnabled(true);
+			}
+			codeTextField.setColumns(10);
+			TextPrompt suggestion = new TextPrompt(MessageBundle.getMessage("angal.common.code.txt"), codeTextField, Show.FOCUS_LOST);
+			suggestion.setFont(new Font("Tahoma", Font.PLAIN, 12));
+			suggestion.setForeground(Color.GRAY);
+			suggestion.setHorizontalAlignment(JLabel.CENTER);
+			suggestion.changeAlpha(0.5f);
+			suggestion.changeStyle(Font.BOLD + Font.ITALIC);
+		}
+		return codeTextField;
+	}
+
+	
+	private void ajustWidth() {
+		for (int i = 0; i < pColumwidth.length; i++) {
+			jTableInventoryRow.getColumnModel().getColumn(i).setMinWidth(pColumwidth[i]);
+		}
+	}
+
+	public EventListenerList getInventoryListeners() {
+		return InventoryListeners;
+	}
+
+	public void setInventoryListeners(EventListenerList inventoryListeners) {
+		InventoryListeners = inventoryListeners;
+	}
+	
+	private JLabel getReferenceLabel() {
+		if (referenceLabel == null) {
+			referenceLabel = new JLabel(MessageBundle.getMessage("angal.common.reference.label"));
+		}
+		return referenceLabel;
+	}
+
+	private JTextField getReferenceTextField() {
+		if (referenceTextField == null) {
+			referenceTextField = new JTextField();
+			referenceTextField.setColumns(10);
+			if (inventory != null && !mode.equals("new")) {
+				referenceTextField.setText(inventory.getInventoryReference());
+				referenceTextField.setEnabled(false);
+			}
+		}
+		return referenceTextField;
+	}
+	
+	private JLabel getLoaderLabel() {
+		if (loaderLabel == null) {
+			ImageIcon icon = new ImageIcon("rsc/icons/oh_loader.GIF");
```

Comment:
the icon filename seems to be lowercase also in the extension part. 

Anyway, do we really need an animated icon in a panel? It would be the first of this genre in Open Hospital...

Reply:
Actually I don't see it in the screenshot you provided
![image](https://github.com/informatici/openhospital-gui/assets/2938553/a82ff375-4df7-4bd3-9c87-59c7580688ed)

Reply:
we see it after cliking on save button the visibility is set to false first. see line 668 in InventoryEdit file

Reply:
I have chang .GIF to .git

Reply:
I see... if so, why to have a different waiting cursor only for this tasks?

Reply:
I don't see another waiting cursor that is why I have use this but like I said if it is not necessary I can remove it.

Reply:
Since version 1.10 every long operation in the EventQueue should gather the OS default waiting cursor, try without and let us know

Reply:
Okay I have see it when I try to open Inventory Management Windows okay I remove it.

Reply:
thanks a lot!